test/cp.js

```diff
@@ -451,6 +451,19 @@ test('-R implies -P', t => {
   });
 });
 
+test('-Ru no longer ignores -u', t => {
+  const [source, dest] = ['foo', 'bar'].map(s => `test/resources/cp/cp-Ru/${s}/file`);
+  // First, update the dest file
+  shell.touch(dest);
```

Comment:
You also need to update the source directory to an older time stamp, right? Otherwise, we'll see race conditions in very-fresh checkouts.

Reply:
In a later comment, you've mentioned checking file contents. That would resolve this issue as well.

Reply:
Yeah, file contents is fine to use here as well.